cairo-tests/src/dict_test.cairo

```diff
@@ -135,6 +135,7 @@ fn test_bool_dict() {
 }
 
 #[test]
+#[ignore]
```

Comment:
why are these tests being ignored? can you add a comment if they aren't working anymore?

Reply:
They're failing. Yesterday I thought it was because of the felt to bool bug but today it doesn't make that much sense to me anymore. I'll check it again just in case.